utils/ms_bot_framework_utils/bot.py

```diff
@@ -44,9 +45,9 @@ def del_conversation(self, conversation_key: ConvKey):
         del self.conversations[conversation_key]
         log.info(f'Deleted conversation, key: {str(conversation_key)}')
 
-    def _init_model(self, server_config: dict):
-        model = Model(server_config)
-        return model
+    def _init_agent(self):
+        agent = self.agent_generator()
+        return agent
```

Comment:
is `_init_agent` method necessary?

Reply:
For now - yes. Placed TODO to remind about further possible refactoring.